extensions/amp-story/1.0/amp-story.js

```diff
@@ -1675,6 +1688,9 @@ export class AmpStory extends AMP.BaseElement {
     return this.getPageById(nextPageId);
   }
 
+  /*
+  * Gives unsupported browser state
```

Comment:
Thanks for improving our documentation! Please merge this into the `/** ... */` block below.